### DISCUSSION TITLE
Where to put estimation model runs? plus ceattle run

### DISCUSSION THREAD

**Discussion by grantdadams** [NOAA-EDAB/ms-keyrun#87] [General] [answered]
Better to put on developer's repos or on the ms-keyrun repo?

CEATTLE data and runs are here:
https://github.com/grantdadams/WGSAM-CEATTLE-Skill-Test

**Comment by vbartolino** [comment, marked as answer]
There may be better ways to organise this, and I would be happy to learn, but at the moment in my opinion estimation teams work on the models in their respective repos (thanks to share the link for CEATTLE, here is that one for gadget2 https://github.com/gadget-framework/wgsam-skill-assessment/tree/main)

When the estimation teams have sharable (not necessarily final) procedures on data/model runs/results those can be moved to the wgsam repo (everyone make a subdirectory with the model name) in here:
https://github.com/ices-eg/wg_WGSAM/tree/master/SkillAssessment

What do you think?
